verifier/src/main/java/io/syndesis/verifier/v1/MetadataEndpoint.java

```diff
@@ -43,23 +43,28 @@ public MetadataEndpoint(final String connectorId, final MetadataAdapter<?> adapt
         return new DefaultCamelContext();
     }
 
-    /* default */ final SyndesisMetadata<?> fetchMetadata(final Map<String, Object> properties) {
+    /* default */ final SyndesisMetadata<?> fetchMetadata(final String actionId, final Map<String, Object> properties) {
         try {
             final CamelContext camel = camelContext();
             camel.start();
 
             try {
-                final MetaDataExtension metadataExtension = camel.getComponent(connectorId, true, false)
+                //TODO Kurt
+                String componentId = connectorId;
+                if ("sql".equals(connectorId)) {
```

Comment:
The SQL actions have their own verifier code, which is different from the Stored Procedure, so the actionId is used to find the component. Should we add a config param that governs this toggle?